101-hdinsight-linux-ssh-publickey-metastore-vnet/azuredeploy.json

```diff
@@ -158,8 +158,9 @@
               "hive_database_name": "[parameters('existingHiveMetastoreDatabaseName')]",
               "hive_database_type": "mssql",
               "hive_existing_mssql_server_database": "[parameters('existingHiveMetastoreDatabaseName')]",
-              "hive_existing_mssql_server_host": "[parameters('existingHiveMetastoreServerName')]",
-              "hive_hostname": "[parameters('existingHiveMetastoreServerName')]"
+              "hive_existing_mssql_server_host": "[concat(parameters('existingHiveMetastoreServerName'),'.database.windows.net')]",
```

Comment:
use reference() to get the hostname - you cannot hardcode it...

Reply:
fixed I think